frontend/components/KanbanBoard/index.tsx

```diff
@@ -203,15 +206,15 @@ export default function KanbanBoardNew({
     container.items.push({
       id,
       title: itemName,
-      status: newIssue.status,
+      status: omitDeep(newIssue.status, '__typename'),
```

Comment:
I guess this is ok for now but might I might consider adding omitDeep(newContainers) in the setPageState or whatever so its called only once or whatever. But since this adds 1 at a time i guess its fine. Just leaving a note for my own thoughts.